po/uk.po

```diff
@@ -0,0 +1,23 @@
+UKRAINIAN TRANSLATION.
```

Comment:
This was wrong, is missing a leading `#` character

Will make another release :disappointed:

Reply:
Fixed in https://github.com/mariospr/text-scaler-gnome-shell-extension/commit/aee31f924424b18b3301f2d96ec4df7c6dc35303